src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1306,6 +1316,24 @@ namespace winrt::TerminalApp::implementation
             Automation::AutomationProperties::SetHelpText(splitTabMenuItem, splitTabToolTip);
         }
 
+        Controls::MenuFlyoutItem closePaneMenuItem = _closePaneMenuItem;
+        {
+            // "Close Pane"
+

```

Comment:
```suggestion
```

Reply:
Makes sense, cleaner - committing suggestion. I've never seen this on GitHub as a possibility... that's pretty neat that you can "Commit suggestion."